hs/src/Reach/EditorInfo.hs

```diff
@@ -9,10 +9,19 @@ import Reach.AST.Base
 import Reach.AST.SL
 import qualified Reach.Eval.Core as C
 
+customConfig :: A.Config
+customConfig = A.Config {
+  confIndent = A.Spaces 4,
```

Comment:
4!?!?!?!!? 2, Hamir. We're not animals ;)

Reply:
Ha, [b033753](https://github.com/reach-sh/reach-lang/pull/644/commits/b03375307b10d03d27323a28828d4ff9274c60d3) fixes this. :)